pocs/tests/conftest.py

```diff
@@ -24,48 +19,39 @@ def pytest_collection_modifyitems(config, items):
     Certain tests should only be run when the appropriate hardware is attached.
     These tests should be marked as follows:
 
-    `@pytest.mark.hardware`: Run all hardware tests
     `@pytest.mark.with_camera`: Run tests with camera attached
```

Comment:
Append period to each.

Reply:
The periods annoy you so much you didn't notice they all say "camera" :)